examples/sparsestream/server/data.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
can you give this a more specific name than data? looks like it is specifically stock ticker/tracking profiles